src/sage/modular/multiple_zeta_F_algebra.py

```diff
@@ -706,7 +755,7 @@ def homogeneous_to_vector(self):
             a, b = next(iter(self))[0]
             N = 2 * a + sum(int(x) for x in b)
             return vector(BR, [self.coefficient(b)
-                               for b in basis_f_iterator(N)])
+                               for b in basis_f_iterator(N, self.parent()._start)])
```

Comment:
```suggestion
                               for b in basis_f_iterator(N, F._start)])
```
Very micro-optimization.